src/utils/featheryClient/index.ts

```diff
@@ -441,7 +472,18 @@ export default class FeatheryClient extends IntegrationClient {
     }
     if (userId) formData.set('fuser_key', userId);
 
-    return this._fetch(url, { method: 'POST', body: formData });
+    if (navigator.onLine) {
```

Comment:
const options = { method: 'POST', body: formData };
if (navigator.onLine) return this._fetch(url, options);
else {
  const request = new Request(...
}